### DISCUSSION TITLE
[Feature] Remove all faces from a photo

### DISCUSSION THREAD

**Discussion by yuval-ziv** [immich-app/immich#10300] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

Sometimes you are in a concert, a club, or just a busy place in general. You don't care about the people in the photo/video because you don't know any of them.
I would love to see a feature that allows you to remove all matched faces from a photo.

### Platform

- [X] Server
- [ ] Web
- [ ] Mobile

**Comment by tygill** [comment]
To add to this, it would be nice to be able to remove all faces from multiple pictures in a bulk selection. (E.g., to allow rescanning faces in those pictures with updated clustering.)

**Comment by bo0tzz** [comment]
Closing in favour of #13868, which is more active.
